atmospherex_activity_base/README.md

```diff
@@ -22,7 +22,7 @@ class MyClass(BaseActivityCustomCode):
 The pip command is:
 
 ```shell script
-pip install git+ssh://git@github.com/ambiata/atmospherex.git@{tag}#subdirectory=atmospherex_activity_base
+pip install git+ssh://git@github.com/ambiata/atmosphere-python-sdk.git@{tag}#subdirectory=atmospherex_activity_base
```

Comment:
```suggestion
pip install git+https://github.com/ambiata/atmosphere-python-sdk@{tag}#subdirectory=atmospherex_activity_base
```
Doc https://pip.pypa.io/en/stable/reference/pip_install/#git
Like that we will not use ssh protocol to download the package, as it can be blocked by clients.

Reply:
Ah yes I did it this way on the iag total loss repo, but just did a simple substitution here, good point